nextflow.config

```
@@ -586,7 +601,11 @@ profiles {
 
         docker {
            fixOwnership = true
-           enabled = true
+   	   enabled = params.apptainer == true ? false : true
```

```suggestion
   	   enabled = params.apptainer != true
```
I find this form a bit quicker to read